src/case_gather/cron_jobs.py

```diff
@@ -15,8 +15,9 @@ class GatherCases(CronJobBase):
     def do(self):
         try:
             SentryLogger.warning('Starting case gather')
-            parliament_session = ParliamentSession.objects.latest('created')
-            CaseGatheringService.update_cases_by_session_number(parliament_session)
+            parliament_sessions = ParliamentSession.objects.all().order_by('-id')[:3]
```

Comment:
does it make sense to pull `3` into a variable and share between the Cron jobs?

Something like `PARLIAMENTS_TO_LOOK_AT = 3` ?

Reply:
Yes that is a good idea

Reply:
Ill implement and test this next tuesday